src/interactions/connector.ts

```diff
@@ -61,6 +61,7 @@ interface ConfigParameterTypes {
     onSelectedLayoutIdChanged: (layoutId: string) => void;
     onLayoutsChanged: (layouts: string) => void;
     onConnectorEvent: (state: string) => void;
+    onZoomChanged: (scaleFactor: string) => void;
```

Comment:
It is not linked to the `zoomChanged` from the editor (at the bottom of this file).